src/program/lwaftr/run_nohw/README

```diff
@@ -4,14 +4,15 @@ Usage: run-nohw --help
        --conf, -c PATH      Path to the lwAFTR configuration file.
```

Comment:
Uhm. If that's how upstream does it, ok, but I don't especially like it. At least for our code, I'd rather stick to one way and use and document it all over the place.